tools/ts-generator/rpc/rpc_gen_js.py

```diff
@@ -133,26 +149,35 @@
  * @param socket
  * @param size
  * @param fn
+ * @param returnComplete
  */
 const readNextChunk = (
   socket: Socket,
   size: number,
-  fn: ApplyFn
+  fn: ApplyFn,
+  returnComplete?: boolean
 ): Promise<Buffer> => {
   return new Promise<Buffer>((resolve) => {
     socket.once("data", (data) => {
-      if (data.length > size) {
-        readBufferRequest(data.slice(size), fn, socket);
-        return resolve(data.slice(0, size));
+      if (returnComplete === true) {
```

Comment:
```suggestion
      if (returnComplete) {
```

Reply:
this impact the performance, when you don't use the explicit type, js uses coercion to resolve the condition and this process is heavier than  use `===` operator with `true`

Reply:
Can you add a comment to the code please?